src/app/bav/steps.js

```diff
@@ -33,18 +33,23 @@ module.exports = {
   },
   [APP.PATHS.CONFIRM_DETAILS]: {
     controller: confirmDetails,
-    fields: ["retryCount"],
+    fields: ["attemptCount"],
     next: [
       {
-        field: "retryCount",
+        field: "attemptCount",
         value: undefined,
         next: APP.PATHS.DONE,
       },
       {
-        field: "retryCount",
+        field: "attemptCount",
         value: 1,
         next: APP.PATHS.COULD_NOT_MATCH,
       },
+      {
+        field: "attemptCount",
+        value: 2,
+        next: APP.PATHS.DONE,
+      },
```

Comment:
Is there anyway so show error page here if value > 2?

Reply:
if attemptCount > 2 the backend throws a 401 and the user is shown the error screen so don't think it's actually possible for us to get that in the fe?